### PR TITLE
{lib}[foss/2022a] fastai v2.7.10 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by crubb**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/f/fastai/fastai-2.7.10-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['ccef6a185ae3a637efc9bcd9fea8e48b75f454d0ebad3b6df426f22fae20039d']
+
+download_dep_fail = True
+use_pip = True
+
+builddependencies = [('binutils', '2.38')]
+multi_deps = {'Python': ['3.10.4']}
```
Comment (smoors) [inline easybuild/easyconfigs/f/fastai/fastai-2.7.10-GCCcore-11.3.0.eb:18]: please change to `dependencies`?
I think there are also quite a number of dependencies/extensions missing, see file `settings.ini`

File: easybuild/easyconfigs/f/fastai/fastai-2.7.10-GCCcore-11.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/%(namelower)s'],
+}
```
Comment (smoors) [inline easybuild/easyconfigs/f/fastai/fastai-2.7.10-GCCcore-11.3.0.eb:23]: this is already the default sanity check, so can be removed.

**Review by smoors — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/f/fastai/fastai-2.7.10-GCCcore-11.3.0.eb (hunk 1 above)
Comment (crubb) [reply easybuild/easyconfigs/f/fastai/fastai-2.7.10-GCCcore-11.3.0.eb]: Done - And added all dependencies but `neptune-client` from `settings.ini`.

File: easybuild/easyconfigs/f/fastai/fastai-2.7.10-GCCcore-11.3.0.eb (hunk 2 above)
Comment (crubb) [reply easybuild/easyconfigs/f/fastai/fastai-2.7.10-GCCcore-11.3.0.eb]: Done!

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Comment by crubb** [comment]
I was unsure about a few of the development dependencies, e.g. I couldn't really think of the reason for including `PyTorch-Lightning` in that capacity. I have temporarily removed these dependencies, but they can easily be added back if necessary.

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Comment by smoors** [comment]
Going in, thanks @crubb!

**Pull request merged**